src/main/java/org/prebid/server/bidder/model/BidderError.java

```diff
@@ -14,28 +14,30 @@
 
     Type type;
 
+    int code;
```

Comment:
It's not clear why this field is necessary - it duplicates information already available in `type`